### PR TITLE
Medical Safety Alerts flagged as Urgent

### PR DESCRIPTION
**Pull request by Seraphiana**
Updated app to send an urgent flag with email requests for medical
safety alerts. Flag defaulted to nil in other formats.

[trello card](https://trello.com/c/QbYLGjL2/263-send-medical-device-alerts-urgently)

### CODE REVIEW

**Comment by Seraphiana** [comment]
@tijmenb @elliotcm

File: app/models/document.rb [hunk 1]
```diff
@@ -347,6 +347,10 @@ def send_email_on_publish?
     update_type == "major"
   end
 
+  def urgent
```
Comment (tijmenb) [inline app/models/document.rb]: Might be worth to put something like https://github.com/alphagov/specialist-publisher/blob/6646ebe34f03d8ea9537c622edf83c32060ba27d/app/exporters/formatters/abstract_document_publication_alert_formatter.rb#L99-L102 here too, to carry over the knowledge of this behaviour.

**Comment by tijmenb** [comment]
👍 looks good

File: app/models/document.rb (hunk 1 above)
Comment (Seraphiana) [reply app/models/document.rb]: good point. Is that clearer?

Comment (tijmenb) [reply app/models/document.rb]: 👍 excellent!

**Pull request merged**